apps/prairielearn/src/schemas/schemas/infoAssessment.json

```diff
@@ -284,75 +333,69 @@
         "questions": {
           "description": "Array of questions in the zone.",
           "type": "array",
-          "minItems": 1,
-          "items": {
-            "$ref": "#/definitions/zoneQuestion"
-          }
+          "items": { "$ref": "#/definitions/ZoneQuestionJsonSchema" },
+          "minItems": 1
         },
         "advanceScorePerc": {
-          "$ref": "#/definitions/advanceScorePerc"
+          "description": "Minimum score percentage to unlock access to subsequent questions",
+          "$ref": "#/definitions/AdvanceScorePercJsonSchema"
         },
         "gradeRateMinutes": {
           "description": "Minimum amount of time (in minutes) between graded submissions to the same question.",
           "type": "number",
           "minimum": 0
         },
         "canSubmit": {
-          "description": "A list of group role names that can submit questions in this zone. Only applicable for group assessments.",
+          "description": "A list of group role names that can submit questions in this assessment. Only applicable for group assessments.",
```

Comment:
I think this and similar changes are actually regressions. These properties live at different levels in the JSON hierarchy and have different meanings at each level.

Reply:
Fixed in [0ffc275](https://github.com/PrairieLearn/PrairieLearn/pull/11024/commits/0ffc275517f833b3c018439522c3485474571243). Should really do a followup PR since doing schema overrides is really hard.